ormar/queryset/queryset.py

```diff
@@ -1111,7 +1114,15 @@ async def bulk_update(  # noqa:  CCR001
 
         columns = [self.model.get_column_alias(k) for k in columns]
 
+        # on_update_fields = {
```

Comment:
please either update the code or remove it -> we don't want commented out code in the code base